signin.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="kor">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>signin</title>
+  <link rel="stylesheet" href="css/signin-style.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <img src="img/Linkbrary_big.png">
+    </div>
+    <div class="sign-up">
+    회원이 아니신가요?
+      <a href="/signup.html">회원 가입하기</a> 
+    </div>
+  </header>
+  <main>
+    <form class="form">
+      <div class="login">
+          <label for="email">이메일</label>
+        <input class="email" id="email" name="email" placeholder="이메일을 입력하세요.">
+      </div>
+      <div class="login">
+          <label for="password">비밀번호</label>
+        <div class="passinput">
+          <input class="password" id="password" name="password" type="password" placeholder="비밀번호를 입력하세요.">
+        </div>
+        <div class="eye">
+          <img src="img/eye.png">
+        </div>
+      </div>
```

Comment:
[중요도: 높음] 웬만해서 html구조에 대해서는 피드백을 줄이고 있으나 이 경우는 조금 더 도움이 될 것 같아 남깁니다.
```suggestion
      <label id="password">
          <span>비밀번호</span>
          <input class="password" id="password" name="password" type="password" placeholder="비밀번호를 입력하세요.">
          </label>
          <img src="img/eye.png" class="eye">
```

이렇게만 해도 element수를 많이 줄여볼 수도 있어보입니다.
label안에 감싸는 방식으로도 줄여볼 수 있으니 이렇게 해보는 것도 좋아보입니다 :)

이견 있으시다면 말씀해주세요...!

Reply:
넵! 조언 감사합니다. 간단하게 바꿔보려 했는데 레이아웃이 꼬여서ㅜㅜ  이번주차 과제를 우선 끝낸 후 수정해보겠습니다.>!